README.md

```diff
@@ -1,6 +1,6 @@
 # The Sixie Opinionated Django Cookiecutter
 
-This is a work in progress, but it is heavily inspired by the work
+This is a work in progress project for jumpstarting production-ready Django projects. It is heavily inspired by the work
 of [cookiecutter-django](https://github.com/pydanny/cookiecutter-django)
 and Daniel Roy Greenfeld. It also borrows heavily from the ideas in the
```

Comment:
All projects are in progress. I propose we simplify this too:
```
This project enables jumpstarting production-ready Django projects. The work of [cookiecutter-django](https://github.com/pydanny/cookiecutter-django)
and Daniel Roy Greenfeld heavily inspires it. It also borrows ideas from PyPA [warehouse](https://github.com/pypa/warehouse) since it provides a fantastic developer onboarding experience on a complex project.
```